projects/AutowareCenterPoint/datasets/tier4_dataset.py

```diff
@@ -0,0 +1,114 @@
+from os import path as osp
+import os
+from typing import Callable, List, Union
+
+import numpy as np
+
+from mmdet3d.registry import DATASETS
+from mmdet3d.structures import LiDARInstance3DBoxes
+from mmdet3d.structures.bbox_3d.cam_box3d import CameraInstance3DBoxes
+# from .det3d_dataset import Det3DDataset
+from mmdet3d.datasets.nuscenes_dataset import NuScenesDataset
+
+
+@DATASETS.register_module()
+class Tier4Dataset(NuScenesDataset):
```

Comment:
```suggestion
class T4Dataset(NuScenesDataset):
```

Reply:
(Thank you for your contribution :pray: )

@kaancolak Please make the dataset name `T4Dataset` consistent throughout the repository, not only the variable names but also the file names as well.